docs/changelog.md

```diff
@@ -6,6 +6,40 @@ sidebar_label: API Changelog
 
 ⚠ Consider subscribing to our [**API Newsletter**](http://eepurl.com/g2EiC1) to be notified about upcoming API changes in the future.
 
+## Version 1.14
+
+### Manifest API: New ticketAction type delete`
+
+A new `delete` ticketAction type has been introduced for the [Manifest API](general/manifest-api). This means that a dedicated delete button can be availble to tickets. The action type `delete` should be added to the ticketActions array when updating the channel using the [Manifest API](general/manifest-api). 
+
+The delete action type accepts one kind of config: `{ forceTagging: true }` . Any other action config given will throw an error.
+As with other ticketActions the delete type requires an `id` and `labels` for the 3 supported languages: 'en', 'de', 'fr'.
+
+- webhooks: If configured, the delete action will trigger webhooks. 
+- action reset: The delete action can be reset using the [Ticket API's error handling](inbox/ticket-api#error-handling) route where `:actionId` will be `delete`.
```

Comment:
> where `:actionId` will be `delete`.

this is not accurate, because `:actionId` is an id of the action, not its type. In the example below it would be `delete-id`

Reply:
Fixed